### ISSUE TITLE
[REQ] Environment variable for assets base URL

### ISSUE THREAD

**Issue by tk1114632** [ianlucas/cs2-inventory-simulator#138]
Hi ianlucas,

Thank you for the amazing project.
I'm wondering if there's any way to run the project with asserts being local on server. If I'm understanding correctly, the project is using external module or CDN content for asserts such as images of items. However because of some local Internet issue, I would like that the website use the asserts from local directory. Is it possible?

BTW, since the update bring then new stickers, is there any way to add the new stickers?

Thanks, and appreciate your working.

**Comment by ianlucas** [comment]
Yeah, currently items details and images are scraped and stored in ianlucas/cs2-lib repo. We use Statically to serve the repo for getting the assets.

Serving the assets locally would mean downloading the images and placing them in the assets folder. I think I can add a environment variable for allowing that, but keep in mind it'll require a lot of space for hosting the images.

The new stickers will be added tomorrow once I publish a new version for cs2-lib. If you want to have them as soon as possible you'd need to fork this repo and cs2-lib to update them yourself.

**Comment by tk1114632** [comment]
Hi, thanks for answering so fast.

I tried to download the cs2-lib files, and extract them into **<root_dir>/node_modules/@ianlucas/cs2-lib/assets/locallizations** on the server. (Seems like '**localizations**' is the only directory existed already). Re-build the project, and run. But no new stickers added still.

**Comment by ianlucas** [comment]
You need to build cs2-lib repo and copy all the files generated in `dist` there, then I think it'll work. The app loads the items from `src/items.js` file.

**Comment by tk1114632** [comment]
Great, that works most well, except that the images couldn't be loaded normally. I guess reason was because the CDN content hasn't been updated. 
![image](https://github.com/user-attachments/assets/bb338477-fd92-442b-8833-ebf5ffd7b1ba)


Last question here: is there any temp fix for this? Maybe I can add the images to local directory for now or do I have to wait for CDN update?
